### ISSUE TITLE
Instagram Version out of Date

### ISSUE THREAD

**Issue by pytexelite** [behdadahmadi/instamaker#5]
Hello, I am getting this error:

> {u'Update App to Sign Up': [u'Update Instagram in the app store to sign up. Your version of the app is out of date.']}

Is this because the key is wrong? Please help. I changed hardly anything from the original script.

**Comment by behdadahmadi** [comment]
@pytexelite Hello,yes the script does not work anymore due to Instagram API changes.

**Comment by behdadahmadi** [comment]
@pytexelite If I find a solution,I will change the script to work.

**Comment by justcallmekoko** [comment]
@pytexelite @behdadahmadi I have been working on a modified version of your code here.
I found out when this attempts to create an account, it will create it but the account is automatically disabled.
I started spamming account creation with randomly generated usernames and emails and noticed every now and then it would attempt to make an account and IT WOULD ACTUALLY WORK.

So it would make maybe a few dozen account that get disabled automatically and then out of no where I would get a success response. Sure enough I went and logged into the account.

Edit: I am also using a list of 566 proxies

**Comment by pytexelite** [comment]
Yeah, I finally got it to somehow work where every now and then i would get
an account that was not disabled. Now I think I can do it consistently. I
know that you can't have the same emails for multiple accounts with +1 or
+2 like sam.t.marriott+1@gmail.com, that will be disabled. Also you have to
use the SAME IP for each request when signing up for an account, not like
rotating proxies on each HTTP request.

**Comment by justcallmekoko** [comment]
@pytexelite How is it you managed to create accounts consistently without them being disabled?
I am using the same proxy for each request when creating an account and I only switch to a new proxy when I have finished trying to create one account and try to make another.

I generate about 100 disabled accounts before I get 1 good one.
